coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
please add `coverage/` to your .gitignore, clear the git cache, commit and push up.

https://devconnected.com/how-to-clear-git-cache/